pyproject.toml

```diff
@@ -9,6 +9,7 @@ include = '''
     | tests
     | extra
   )/.*\.py$
+  | extra/changelog-generator/changelog-generator
```

Comment:
I thought I had already included it, as there is the `extra` folder above. What I didn't realize is that the regex only matches `.py` files...

Anyway, thanks!

Reply:
I noticed :)